clarity_ext/domain/process.py

```diff
@@ -4,10 +4,11 @@
 class Process(DomainObjectWithUdfMixin):
     """Represents a Process (step)"""
 
-    def __init__(self, api_resource, process_id, technician, udf_map, ui_link):
+    def __init__(self, api_resource, process_id, technician, udf_map, ui_link, instrument):
         super(Process, self).__init__(api_resource, process_id, udf_map)
         self.technician = technician
         self.ui_link = ui_link
+        self.instrument = instrument
 
     @staticmethod
```

Comment:
Currently, we don't use instrument on step level at snpseq. Should we include this anyway?

Reply:
I think it makes sense since this is available on this level in Clarity. It doesn't mean we need to use it there.